docs/install.md

```diff
@@ -18,12 +18,9 @@ First, install Docker version 1.3 or greater:
 
 ### Install Compose
 
-To install Compose, run the following commands:
+To install Compose, run the following command:
 
-    curl -L https://github.com/docker/compose/releases/download/1.2.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
-    chmod +x /usr/local/bin/docker-compose
-
-> Note: If you get a "Permission denied" error, your `/usr/local/bin` directory probably isn't writable and you'll need to install Compose as the superuser. Run `sudo -i`, then the two commands above, then `exit`.
+    sudo bash -c "curl -L https://github.com/docker/compose/releases/download/1.2.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose && chmod +x /usr/local/bin/docker-compose"
```

Comment:
Many users (such as [Homebrew](http://brew.sh/) users) will have `/usr/local` owned by a non-root user, but this approach will result in the actual `docker-compose` file being owned by `root` regardless of the ownership of the rest of the directory. That can cause confusion down the line.

I acknowledge that software security is fully a PITA, but does Docker/Compose really want to encourage users to use `sudo` to install it by default? Users who don't want to understand these issues should simply use a package manager like `pip` or homebrew itself` to install `docker-compose`.